data/singhmohit14072002.json

```
@@ -4,6 +4,15 @@
   "type": "personal",
   "bio": "OpenSource  | DevOps | Docker | Blogger **Hashnode**",
   "avatar": "https://github.com/singhmohit14072002.png",
+  "tags": [
+    "Open Source",
+    "DevOps",
+    "AWS", 
+    "Auzur",
```

I think this may be a typo and should read 

```suggestion
    "Azure,
```

As this is not a blocker I will merger it.  Should you wish to correct this then please raise another PR.

---

- ok